src/modules/display-comments.js

```diff
@@ -0,0 +1,43 @@
+const commentApi = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/4dyq58hiPupb0utce3OK/comments';
+
+export const fetchComments = async (id) => {
+  const response = await fetch(`${commentApi}?item_id=${id}`);
+  const data = await response.json();
+
+  if (!response.ok) {
+    return null;
+  }
+  return data;
+};
+
+// eslint-disable-next-line
+const createCommentHtml = ({ creation_date, comment, username }) => {
+
+  // eslint-disable-next-line
+  const creationDate = creation_date;
+  const commentli = document.createElement('li');
+  commentli.className = 'comment-li';
+  commentli.innerHTML = `
```

Comment:
Kindly implement the user to be able to input the comments and then to be displayed under comments as the Project requirement state; unable to see the inputs of the message
![image](https://user-images.githubusercontent.com/67240581/204009961-d1dfff81-c55a-4255-8cda-b34efebf51b5.png)

Reply:
Hi @ernestkorir ,
The ability to add comments is a feature that needs to be implemented after this.
I am going to work on it on the next branch.